_schemaV2.graphql

```diff
@@ -1359,6 +1359,9 @@ type Artwork implements Node & Searchable & Sellable {
   # canvas_. (This should not be confused with the artwork attribute called
   # `category`, which is commonly referred to as "medium" or "medium type")
   medium: String
+
+  # Represents the "**medium**" of an artwork, such as painting_
+  mediumType: MediumType
```

Comment:
Noticing that we have duplication in the schema now between `artwork.category` and `artwork.mediumType.name` — which I worry will add to the already existing [confusion around medium vs category](https://github.com/artsy/metaphysics/pull/2868). 

I guess we can't simply nest these documentation fields under `artwork.category` since that's already a string scalar, right?

Should we actually deprecate `artwork.category` 😬 ?

Curious about others' thoughts. (Also I'd vote for keeping some version of the new [high-fidelity description](https://github.com/artsy/metaphysics/blob/0111aca687dca81b10d65f6e23b8a51639979486/_schemaV2.graphql#L1239-L1241) on `artwork.mediumType` if we retain this field.)

Reply:
> Should we actually deprecate artwork.category 😬 ?

I'd be in favor of deprecating `category` in favor of `mediumType -> name`.

> Also I'd vote for keeping some version of the new verbose description on artwork.mediumType if we retain this field, since mediumType vs medium confusion is still very possible.)

For sure, will add that verbose description 👍

Reply:
Marked the Artwork `category` field as deprecated and updated the new field description 👍